base/field_abilities/Wave.ts

```diff
@@ -27,14 +27,21 @@ export class Wave {
     }
 
     resize(width: number, height: number) {
-        this.width = width;
-        this.height = height;
+        this.width = Math.round(width);
+        this.height = Math.round(height);
         this.half_height = this.height >> 1;
         this.half_width = this.width >> 1;
         this.frequency = 2 / this.half_width;
         this.bmp?.resize(this.width, this.height);
     }
 
+    reposition(x: number, y: number) {
```

Comment:
We don't need to change wave position since it doesn't move while pound is cast. If you're having problems to scale it, set the anchor point to 0.5: `your_wave.sprite.anchor.setTo(.5, .5)`.

Reply:
While the wave is being resized during the tween animation It would "move" from its position, instead of just expanding from the center

Reply:
Se this anchor property to solve this issue.